src/luna/pathology/cli/dsa_annotation_etl.py

```diff
@@ -29,28 +30,25 @@ def cli(
     dsa_endpoint: str = "???",
     collection_name: str = "???",
     annotation_name: str = "???",
-    username: str = "???",
-    password: str = "???",
+    username: str = "${oc.env:DSA_USERNAME}",
```

Comment:
Shouldn't this be

    username: str = os.getenv("DSA_USERNAME", "???>"),
    password: str = os.getenv("DSA_PASSWORD", "???"),

I'm not sure what oc.env is, but this is the normal pythonic way to use environment variable values.

Reply:
It's the way to do it using OmegaConf which is the library we're using for configuration. See https://omegaconf.readthedocs.io/en/2.3_branch/custom_resolvers.html#built-in-resolvers

Reply:
You can also set it to fallback to ??? but the point of that is that it throws an error if missing, which it no longer does after the resolution.

Reply:
Ok, makes sense.